pkg/backends/influxdb/backend.go

```diff
@@ -137,6 +137,30 @@ func (b *Backend) getMetricPoints(
 		points = append(points, point)
 	}
 
+	for _, sleepAnalysis := range metric.SleepAnalyses {
+		startFields := make(map[string]interface{})
+		endFields := make(map[string]interface{})
+
+		// Add qty if set
+		if sleepAnalysis.Qty != 0 {
+			endFields["qty"] = float64(sleepAnalysis.Qty)
+		}
+
+		// tags
+		tags["source"] = sleepAnalysis.Source
+		tags["value"] = sleepAnalysis.Value
```

Comment:
Just a point of discussion, no need to make any changes here:

I just noticed that we use `tags` for SleepAnalyses, unlike aggregated sleep analysis data (which stuffs everything into `fields`). Not sure if we should change aggregated sleep analysis datapoints to also write to tags for `sleepSource`, etc. fields? What do you think?

Reply:
Yes, let's do this in a new PR!

---

I was actually using `sleepStart` and `sleepEnd` fields to perform some funky computation in order to visualize the average time I go to sleep every day:

![image](https://user-images.githubusercontent.com/9884746/201977888-796c58f5-9a4a-4a6f-9bfc-f21e244c4c8c.png)

This involved an extremely complex Flux query, and it doesn't really handle cases crossing the midnight boundary, which results in some rather dubious data when I changed the time range:

![image](https://user-images.githubusercontent.com/9884746/201978509-bc99ae6d-5860-4b2d-98b6-aa2f6aead8b4.png)

I imagine there might be a better way to support this if it was properly supported by the ingester. But for now, I am also okay to introduce a breaking change, since I don't really rely on this data now!

Reply:
Interesting use of the data. I'm not sure how to support an average going-to-sleep-time with the fields I proposed. Perhaps we could include a time-of-day field for "went to sleep" and "woke up" in minutes/seconds/millis since midnight or something? If that's of interest to you, feel free to edit/comment on #11.

Reply:
I think it's pretty hard to compute an average for time-of-day values. A more meaningful metric could be *deviation from expected sleep time on a given day of week*, I think. Let's ignore this use case for now!